### PR TITLE
#103 have added in license expression to all projects

### LINKED ISSUE #103: Nuget package should have addtional metadata
### Summary
I wish for the nuget packages to have the licence expression property set correctly and the repository url

### Details

The licence expression property should be set to the correct licence type I.e. Apache-2.0 as this will enable analysis of licences in use to occur in external tools & the license type will be shown in Nuget etc. By providing the repo url it will make it easier to contribute and can be used by source-link.

### PR DESCRIPTION
**Pull request by thompson-tomo**
Closes #103

### CODE REVIEW

File: src/jmespath.net.parser/jmespath.net.parser.csproj [hunk 1]
```diff
@@ -28,21 +28,21 @@
...
   <ItemGroup>
     <ProjectReference Include="..\jmespath.net.interop\jmespath.net.interop.csproj" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-	  <!-- Use GplexBuffers and ShiftReduceParserCode from the net13 folder instead, which are modified for use of netstandard 1.3 -->
-	  <Compile Remove="ShiftReduceParserCode.cs" />
-	  <Compile Remove="GplexBuffers.cs" />
-	  <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+      <!-- Use GplexBuffers and ShiftReduceParserCode from the net13 folder instead, which are modified for use of netstandard 1.3 -->
```
Comment (springcomp) [inline src/jmespath.net.parser/jmespath.net.parser.csproj:39]: Although reformatting is needed, please leave that out for another pull request, if any.

File: src/jmespath.net/jmespath.net.csproj [hunk 2]
```diff
@@ -26,13 +26,13 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
-	<PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```
Comment (springcomp) [inline src/jmespath.net/jmespath.net.csproj:29]: Likewise, please do not mix concerns in a single pull request.

**Review by springcomp — CHANGES_REQUESTED** [review]
Looks good with feedback.

File: src/jmespath.net/jmespath.net.csproj (hunk 2 above)
Comment (thompson-tomo) [reply src/jmespath.net/jmespath.net.csproj:29]: Will look at reverting the whitespace tomorrow.

**Review by springcomp — APPROVED** [review]
